athena_udf/__init__.py

```diff
@@ -1,10 +1,18 @@
 import base64
+from concurrent.futures import ThreadPoolExecutor, as_completed
+from typing import Optional
 from uuid import uuid4
 
 import pyarrow as pa
 
+from .utils import get_chunks
```

Comment:
nit: is the . necessary? I think that `from .utils import get_chunks` works as well.

Reply:
nice catch, I've fixed the library files structure